index.html

```diff
@@ -14,256 +14,258 @@
-                    </ul>
-                </div>
-            </div>
-            <footer class="submit">
-                <button type="submit">Get my resume</button>
-            </footer>
-        </section>
-        <section class="Skills">
-            <div class="Languages">
-                <div class="lang">
-                    <h3>Languages</h3>
-                </div>
-                <div class="script">
-                    <img src="/photos/javasvript.png" alt="">
-                    <p>JavaSript</p>
+            <section class="Skills">
+                <div class="Languages">
+                    <div class="lang">
+                        <h3>Languages</h3>
+                    </div>
+                    <div class="script">
+                        <img src="/photos/javasvript.png" alt="">
+                        <p>JavaSript</p>
+                    </div>
+                    <div class="script">
+                        <img src="/photos/html5.png" alt="">
+                        <p>HTML</p>
+                    </div>
+                    <div class="script">
+                        <img src="/photos/css3.png" alt="">
+                        <p>CSS</p>
+                    </div>
                 </div>
-                <div class="script">
-                    <img src="/photos/html5.png" alt="">
-                    <p>HTML</p>
+                <div class="frame">
+                    <p>Frameworks</p>
+                    <img src="/photos/separator.png" alt="">
                 </div>
-                <div class="script">
-                    <img src="/photos/css3.png" alt="">
-                    <p>CSS</p>
+                <div class="ski">
+                    <p>Skills</p>
+                    <img src="/photos/separator.png" alt="">
                 </div>
+            </section>
+        </div>
+    
+        <div class="l-container">
+            <div class="contact">
+                <h1>Contact me</h1>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+                coding. I’d love to help with it</p>
             </div>
-            <div class="frame">
-                <p>Frameworks</p>
-                <img src="/photos/separator.png" alt="">
-            </div>
-            <div class="ski">
-                <p>Skills</p>
-                <img src="/photos/separator.png" alt="">
+            <div class="form">
+                <form action="GET">
+                    <input type="text" name="name" placeholder="Merianne">
+                    <br>
+                    <input type="email" name="email" placeholder="natajayanj14@gmail.com">
+                    <br>
+                    <textarea name="name"  cols="38" rows="10">Write your message here</textarea>
+                    <div class="submit">
+                        <button type="submit">Get It Touch</button>
```

Comment:
- [ ] I kindly suggest that you follow the template design for your button

| Your Button Design | Template Design Button |
| ------------- | ------------- |
| ![Screenshot from 2022-05-04 15-14-27](https://user-images.githubusercontent.com/52934128/166700343-86cfb98a-4e2c-4c72-9a8c-c1d8cba96fda.png) | ![Screenshot from 2022-05-04 15-13-06](https://user-images.githubusercontent.com/52934128/166700134-7963bbca-90fc-4ebc-995d-f366056234a1.png) |

- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```